index.html

```diff
@@ -221,7 +221,28 @@ <h4>Skills</h4>
       </div>
     </div>
   </section>
+  <section><img src="./images/SideImage.svg" id="nav-image" alt="" />
+    <div id="contact-me">
+      <h2>Get Started</h2>
+      <p>
+        I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+      </p>
+      <form action="https://formspree.io/f/mqkovwwj" method="POST">
+        <div class="form">
+          <label for="name">Your name</label><br>
+          <input id="name" type="text" name="name" placeholder="Donice Ubaru" maxlength="30" required>
+          <label for="email">Your email</label><br>
+          <input id="email" type="email" name="email" placeholder="doniceubaru1@gmail.com" required>
+          <label for="message">Your mesaage</label><br>
+          <textarea id="message" type="text" name="message" rows="4" cols="50" placeholder="Enter yout new message here"
+            maxlength="500" required></textarea>
+        </div>
 
+        <button type="submit" id="submit-button">Start Collaboration</button>
+      </form>
+    </div>
+  </section>
+  
   <footer>
     <a href="/">Get my Resume<span class="material-symbols-outlined">
         open_in_new
```

Comment:
- [ ] As requested by the previous reviewer, Based on the Figma design for your template, the footer should occupy the bottom part of the page. Your footer design has white space at the bottom of the page. Kindly adjust this. :+1:

  ### Your design
  ![image](https://github.com/donice/Portfolio-Microverse/assets/92303891/a449226e-019a-4efc-bf5d-7cdbf263cd9b)
  
  ### Figma design
  ![image](https://github.com/donice/Portfolio-Microverse/assets/92303891/baa48967-10c8-4e63-b446-7db814ad23d8)